### PR TITLE
Support proctored flag

### PR DESCRIPTION
**Pull request by bjrne**
Since I am not enrolled in any proctoring courses on a live platform, I couldn't test this. The dialog descriptions still need the correct texts and a translation. Does this warning already exist somewhere else?

### CODE REVIEW

File: xikolo-ios/ViewControllers/Courses/CourseContentTableViewController.swift [hunk 1]
```diff
@@ -79,6 +79,19 @@ class CourseContentTableViewController: UITableViewController {
         }
     }
 
+    func showProctoringDialog() {
+        let title = NSLocalizedString("This item needs to be proctored", comment: "Shown in proctoring dialog")
+        let message = NSLocalizedString("You can't do this on your mobile device.", comment: "Shown in proctoring dialog")
```
Comment (bjrne) [inline xikolo-ios/ViewControllers/Courses/CourseContentTableViewController.swift]: These descriptions need an update and a translation

Comment (rhlff) [inline xikolo-ios/ViewControllers/Courses/CourseContentTableViewController.swift]: Please take care of these before merging.

Comment (bjrne) [inline xikolo-ios/ViewControllers/Courses/CourseContentTableViewController.swift]: @jprberlin @rhlff I thought you may want to add a more elaborate explanation or hints on what to do here. I will add the translation when the message is ready. Or is this enough?

**Comment by rhlff** [comment]
Please also wait until this feature is merged at our backed, in case something will be changed last minute. I will notify you.

File: xikolo-ios/ViewControllers/Courses/CourseContentTableViewController.swift (hunk 1 above)
Comment (rhlff) [inline xikolo-ios/ViewControllers/Courses/CourseContentTableViewController.swift]: That's why I asked you to take care of it. The text should be more expressive. A good start point could be the qualified certificate page at our web client, to align the wording. For example I'm not sure if the user knows what proctoring means or if this is only an internal term.

**Comment by bjrne** [comment]
@rhlff is this enough? We should still wait for the backend to be merged

**Comment by jprberlin** [comment]
Please rebase, i giess this rebase is related to the New usage of boolean attribute

**Comment by bjrne** [comment]
I will, but @rhlff wants to refactor this in the backend so this must not be merged before that.

**Comment by rhlff** [comment]
I just adjusted the texts a bit, please take care of updating them:
```
"This item needs to be proctored" = "Online-Aufsicht erforderlich";
"In order to receive your booked qualified certificate you have to complete this assignment on a computer with webcam." = "Diese Aufgabe muss unter Online-Aufsicht abgelegt werden, um das von Ihnen gebuchte qualifizierte Zertifikat zu erhalten. Das ist nur an einem Computer mit Kamera möglich.";
```

**Comment by mathebox** [comment]
LGTM

**Pull request merged**